python/paddle/v2/framework/graph.py

```diff
@@ -0,0 +1,45 @@
+import paddle.v2.framework.core as core
```

Comment:
I am not sure what's the name of this module. It will contain `Block`, `Program`, `Var`, and `Operator`.

Since they will cross-reference each other, they can only be in one Python file.

Use `graph.py` temporary, if there is a better name, it will be changed in following PR.